modules/@shopify/checkout-sheet-kit/android/src/main/java/com/shopify/reactnative/checkoutsheetkit/CustomCheckoutEventProcessor.java

```diff
@@ -45,26 +46,44 @@ public CustomCheckoutEventProcessor(Context context, ReactContext reactContext)
 
   @Override
   public void onCheckoutCompleted() {
-    sendEvent(this.reactContext, "completed", null);
+    sendEvent("completed", null);
+  }
+
+  @Override
+  public void onWebPixelEvent(@NonNull PixelEvent event) {
+    try {
+      ObjectMapper mapper = new ObjectMapper();
+      String data = mapper.writeValueAsString(event);
+      sendPixelEvent(data);
+    } catch (IOException e) {
+      Log.e("ShopifyCheckoutSheetKit", "Error processing pixel event", e);
+      e.printStackTrace();
```

Comment:
Should we remove the print stack trace now that we have the log?

Reply:
Does the log include the stack trace by default?